content/python/concepts/threading/terms/local/local.md

```diff
@@ -0,0 +1,58 @@
+---
+Title: '.local()'
+Description: 'Returns a local thread object with data that is specific to that thread.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Async Await'
+  - 'Functions'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+The `.local()` method returns a local thread object with data that is specific to that thread.
+
+## Syntax
+
+```pseudo
+threading.local()
+```
+
+No parameters are used in the `.local()` method.
+
+## Example
+
+Additionally, the object returned by `.local()` can be assigned to a variable, which is then stored in memory:
+
+```pseudo
+import threading
+
+local_thread = threading.local()
+
+print(local_thread)
+```
+
+The output would look like this:
+
+```shell
+<_thread._local object at 0x7fde6ae24200>
```

Comment:
Do we want to add a small bit about how to memory location alphanumeric will differ